department-of-veterans-affairs/abd-vro
# Mcp 2554 - Cleanup unit tests in svc-lighthouse-service

Fix Blood pressure reading test.

## What was the problem?
The blood pressure reading test failed after we added partial BP reading. 

Associated tickets or Slack threads:
 MCP-2554

## How does this fix it?[^1]
The unit test used the test data from Lighthouse FHIR API returned Observation data a while back. The version of HapiFhir library we used now cannot parse/handle BP reading values in that data correctly. After we updated our BP reading to filter out empty value and handle the partial reading, this issue was detected in the unit test. So the test failed.

As Lighthouse FHIR API has apparently changed and it does not return BP reading data in the way as we see the test data we used, replacing the test data with the new Lighthouse FHIR API BP reading data fixes the unit test.

## How to test this PR
- Step 1. Build the app. No unit test failure should happen.
- Step 2[^secrel]


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
